website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+<br>
+1. Elasticity- which allows flexibility capability meaning on demand, flexible in time, predictable and intelligent mixed load handling ability
+2. Resiliency - User’s unperceived High Availability capability, including transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc
+3. Federated and Analytical - Multi-mode federal analysis capabilities, including analysis and integration capabilities for multiple formats and systems, comprehensive analysis functions and excellent interactive analysis performance and experience, parallel computing processing capabilities, compatible interface capability and so on.
+<br><br>
+
+**Third layer**: Data analysis application and visualization application layer
+<br><br>
+Business logic based upon Data Lake analysis service layer may require an integration of data analysis tools on the cloud. All kinds of visualized data analysis products and tools are available to be interacted with users.
+<br>
+Upon the basis of the above cloud data lake analysis architecture, the data federation ecosystem is divided into three layers accordingly, for which Data Lake Analytics plays a key role for serverless cloud-native computing framework:
+<br><br>
+
+![](/img/blog/2020-06-30-data-lake-analytics-blog/dla_three_tier.png)
+
+<br>
+
+Presto is known to be a high-performance, distributed SQL query engine for Big Data. It offers large-scale data analytics with multiple connectors for accessing various data sources. This capability enables the Presto users to further extend some features to build a large-scale data federation service on cloud.
+
+<br><br>
+
+
+
+## Data Lake Analytics
+### Introducing DLA 
+
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud.
```

Comment:
Is it forked from 0.227 permanently or is it continuing to track future updates to PrestoDB?

Reply:
we do plan to stay close with the community with a close contact with the community to understand the latest updates and ongoing projects so we can pickup important features from the community or contribute any bugfix or features back to the PrestoDB.